launcher/ui/MainWindow.cpp

```diff
@@ -1603,26 +1603,36 @@ void MainWindow::setCatBackground(bool enabled)
         QDateTime now = QDateTime::currentDateTime();
         QDateTime birthday(QDate(now.date().year(), 11, 30), QTime(0, 0));
         QDateTime xmas(QDate(now.date().year(), 12, 25), QTime(0, 0));
+
         QString cat;
```

Comment:
Recommend guarding against an invalid CatStyle being specified in the config file. Example:
```cpp
QString cat = "default";
QString catStyleSetting = APPLICATION->settings()->get("CatStyle").toString();

if (catStyleSetting == "Jinx")
{
    cat = "jinx";
}
```
(You could write it in a different way, as long as it has the same effect)